packages/scss/src/commons/base.scss

```diff
@@ -94,6 +94,10 @@
 			font-weight: 600;
 		}
 
+		*:where([tabindex='0']:focus-visible) {
+			@include a11y.focusVisible($borderRadius: var(--commons-borderRadius-M));
+		}
```

Comment:
Not sure about this, there are various instances where I want the focus to display around a parent element rather than the focusable element itself for visual reasons. I would have to override this rule to avoid a double focus effect.

Reply:
You're right, I'll close until I find something better.

Reply:
Thanks!